payload/src/dynamics.ts

```diff
@@ -147,6 +150,27 @@ export function FormSubmit(self: HTMLElement, e: Event) {
     return false
 }
 
+function TriggerLoaders(self: HTMLElement) {
+    // Find component root
+    let root = _LocateRoot({
+        starter: self,
+    })
+    // Find loader elements
+    // Need escape for escaping (for payload wrapper)
+    let loader = root.querySelectorAll('[ssa\\\\:oncall\\\\.display]')
+    loader.forEach(element => {
+        // Check attribute value
+        let loadertype = element.getAttribute('ssa:oncall.display')
+        // Show element, if value is empty
```

Comment:
Just ignore in case of empty value. Don't need default value, because it would become semantically unclear (people who don't know usage will need to check documentation)